tests/FaultyGraphSegmentationTest.java

```diff
@@ -4,7 +4,6 @@
 import sbr.*;
 
 public class FaultyGraphSegmentationTest {
-    @Test
     public void faulty3x3Graph() throws Exception {
```

Comment:
This file, need to have this annotation before each test. Make sure to put this at start of each test and to keep first letter capital.